### PR TITLE
ammending procfile to capture logs

### PR DESCRIPTION
**Pull request by heckj**
Per slack conversation on 1/8 #RackHD, debugging Pasu's issues
really needed to see the logs - and what was missing from them,
so I thought it might be beneficial to capture them in the procfile
setup on a per-run basis, hence this update.

### CODE REVIEW

File: packer/ansible/roles/foreman/files/Procfile [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 # SUDO must be enabled in the terminal before running nf start (or run sudo nf start)
 # See .env for environment variable definitions
-graph: /bin/bash -c "echo running $TASKGRAPHDIR/index.js; cd $TASKGRAPHDIR && exec sudo node index.js"
-http: /bin/bash -c "echo running $HTTPDIR/index.js; cd $HTTPDIR && exec sudo node index.js"
-dhcp: /bin/bash -c "echo running $DHCPDIR/index.js; cd $DHCPDIR && exec sudo node index.js"
-tftp: /bin/bash -c "echo running $TFTPDIR/index.js; cd $TFTPDIR && exec sudo node index.js"
-syslog: /bin/bash -c "echo running $SYSLOGDIR/index.js; cd $SYSLOGDIR && exec sudo node index.js"
+graph: /bin/bash -c "echo running $TASKGRAPHDIR/index.js; cd $TASKGRAPHDIR && exec sudo node index.js | tee /tmp/taskgraph.log"
+http: /bin/bash -c "echo running $HTTPDIR/index.js; cd $HTTPDIR && exec sudo node index.js | tee /tmp/taskgraph.log"
```
Comment (benbp) [inline packer/ansible/roles/foreman/files/Procfile]: Are you intending to aggregate all logs to the same file? If so, perhaps rackhd.log or something is a better name?

Comment (heckj) [inline packer/ansible/roles/foreman/files/Procfile]: I'm a moron

Comment (benbp) [inline packer/ansible/roles/foreman/files/Procfile]: :dancer:

**Comment by heckj** [comment]
ahhhh! damnit - will fix unintended submodule updates

**Comment by benbp** [comment]
:+1:

**Comment by anhou** [comment]
:+1:

**Pull request merged**